app/dashboard/views.py

```diff
@@ -2084,3 +2092,38 @@ def get_kudos(request):
         raise Http404
     mimetype = 'application/json'
     return HttpResponse(data, mimetype)
+
+
+def hackathon(request, hackathon=''):
+    """Handle rendering of HackathonEvents. Reuses the dashboard template."""
+
+    evt = None
```

Comment:
Why pre-declare `evt` here? If it is set in the try: block then its within the function scope as normal, and if that raises an exception, then a 404 is returned.